src/coreclr/vm/eventing/userevents/LinuxTracepoints/.gitattributes

```diff
@@ -0,0 +1,12 @@
+* text=auto
```

Comment:
Is this a copy of 3rd party project? It should live under https://github.com/dotnet/runtime/tree/main/src/native/external, have the .txt file with details where it came from, and be listed in https://github.com/dotnet/runtime/blob/main/src/native/external/cgmanifest.json for component governance.

Reply:
(You can submit the verbatim copy of the project in separate PR to make the integration easier to review. Similar to what's being done in https://github.com/dotnet/runtime/pull/102520 for zlib-ng.)

Reply:
It is a copy of a 3rd party project, I can move it but I won't be able to do any work on this PR until next week.

Reply:
It is now moved to src/native/external